mainapp/tests/test_views.py

```diff
@@ -179,3 +179,57 @@ def test_creation(self):
         self.assertEqual(contributor.district, 'pkd')
         self.assertEqual(contributor.phone, '8893845901')
         self.assertEqual(contributor.address, 'Near Mosque')
+
+class ParseRescueDataTests(TestCase):
```

Comment:
There should be 2 blank line before the start of a class.

Reply:
PEP8 is not being strictly followed here. I guess we can let this go.
@biswaz - what say?

Reply:
Past mistakes should not be taken forward. It's better to write good quality code from the moment to help out future contributors.